src/Servers/IIS/Directory.Build.props

```diff
@@ -1,6 +1,13 @@
 <Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+  
   <PropertyGroup>
     <KestrelSharedSourceRoot>$(MSBuildThisFileDirectory)..\Kestrel\shared\</KestrelSharedSourceRoot>
+    <HostArchDir>$(Platform)</HostArchDir>
+    <HostArchDir Condition="'$(Platform)' == 'Win32'">x86</HostArchDir>
+    <HostArchDir Condition="'$(Platform)' == 'AnyCPU'">x64</HostArchDir>
+    <LibNetHostPath>$(NetCoreTargetingPackRoot)\Microsoft.NETCore.App.Host.win-$(HostArchDir)\$(AppHostPackVersion)\runtimes\win-$(HostArchDir)\native</LibNetHostPath>
```

Comment:
Any chance this could ever change?

Reply:
This is meant to change 'automatically' with every SDK update, that stuff comes from the Directory.props.in magic

Reply:
See the stuff in GenerateProps: [eng/tools/GenerateFiles/GenerateFiles.csproj](https://github.com/dotnet/aspnetcore/pull/39810/files#diff-079db1ca47ec56a5adb02a25bab9b57e1b359f8872eb6d3aea53103b962f84e7)

Reply:
What I mean is that you've hard coded the format of the path to libnethost here, what happens if they start dropping that into a different location?

Reply:
We'd have to update this path whenever we take SDK update, I'd imagine this wouldn't change often as this is the dotnet SDK runtime packs layout

Reply:
Our builds would fail immediately since we use this path to link against libnethost.lib which no longer would be found so its not something we can miss